src/tracer.js

```diff
@@ -34,6 +34,8 @@ export const MAX_ELEMENTS_IN_EXTRA = 10;
 export const LEAK_MESSAGE =
   'Execution leak detected. More information is available in: https://docs.lumigo.io/docs/execution-leak-detected';
 
+Timer.init(process.env.LUMIGO_TRACER_TIMEOUT || 500);
```

Comment:
This should be called before the start each invocation, if you will leave it here its will not reset every invocation (This called now once per process [Container])

Reply:
it doesnt starts the timer just init it to the wanted duration